app/controllers/email_addresses_controller.rb

```diff
@@ -42,7 +42,7 @@ def create
   def update
     respond_to do |format|
       if @email_address.update(email_address_params)
-        format.html { redirect_to @email_address.person, notice: 'Email address was successfully updated.' }
+        format.html { redirect_to @email_address.contact, notice: 'Email address was successfully updated.' }
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.